app/models/courseStatus.py

```diff
@@ -1,4 +1,7 @@
-from app.models import*
+# from app.models import*
+from app.models import baseModel
+from peewee import CharField
```

Comment:
We can leave the previous import statement as is.

Reply:
Now using 'from app.models import*' only.